content/ko/home/required.md

```diff
@@ -0,0 +1,42 @@
++++
+# Custom widget.
+# An example of using the custom widget to create your own homepage section.
+# To create more sections, duplicate this file and edit the values below as desired.
+widget = "featured-section"
+active = true
+date = 2016-04-20T00:00:00
+
+title = "KubeEdge는 완전한 오픈소스이며, 오픈소스 기술 위에서 동작합니다."
+subtitle = ""
+
+# Order that this section will appear in.
+weight = 4
+
+# Showcase personal skills or business features.
+# 
+# Add/remove as many `[[feature]]` blocks below as you like.
+# 
+# Available icon packs and icons:
+# * fas - Font Awesome standard icons (see https://fontawesome.com/icons)
+# * fab - Font Awesome brand icons (see https://fontawesome.com/icons)
+# * ai - academic icons (see https://jpswalsh.github.io/academicons/)
+
+[[featured]]
+  img_src = "kubernetes-200.png"
+  img_width = "100px"
+  name = "쿠버네티스"
+  description = "KubeEdge 프로젝트는 쿠버네티스 기술을 기반으로 하는 개방형 플랫폼을 만들고 있으며 클라우드와 엣지 간의 네트워크, 애플리케이션의 배포 및 메타 데이터 동기화를 위한 기본 인프라 지원을 제공합니다."
+  
+[[featured]]
+  img_src = "mosquitto-200.png"
+  img_width = "100px"
+  name = "모스키토"
+  description = "이클립스 모스키토는 MQTT 프로토콜 버전 3.1 및 3.1.1을 구현한 오픈 소스(EPL / EDL 라이센스) 메시지 브로커입니다. 모스키토는 가볍고 저전력의 단일 보드 컴퓨터에서부터 전체 서버에 이르는 모든 장치에까지 사용하기에 적합합니다."
+  
+[[featured]]
+  img_src = "docker-200.png"
+  img_width = "100px"
+  name = "도커"
+  description = "컨테이너를 사용하면 새로운 응용 프로그램을 빠르게 구축하고 배포 할 수 있습니다. 도커 컨테이너는 소프트웨어와 관련된 종속성들을 소프트웨어 개발을 위해 표준화 된 단위로 묶습니다. 여기에는 코드, 런타임, 시스템 도구 및 라이브러리와 같은 실행에 필요한 모든 것이 포함됩니다."
```

Comment:
Do Kubernetes, mosquitto, and Docker have corresponding Korean translations?

Reply:
There is no Korean equivalent word for 'Kubernetes', 'Mosquitto', 'Docker'.
In case of Kubernetes, a localization project exists, and that project uses the pronunciation of Hangul as it is and translates it as '쿠버네티스','도커'

https://kubernetes.io/ko/docs/contribute/localization_ko/#%ED%95%9C%EA%B8%80%ED%99%94-%EC%9A%A9%EC%96%B4%EC%A7%91

Reply:
Thanks, this is helpful